### PR TITLE
rg-47: Update refresh token logic

### PR DESCRIPTION
**Pull request by Icerock10**

### CODE REVIEW

File: backend/src/common/config/config.package.ts [hunk 1]
```diff
@@ -59,6 +61,15 @@ class Config implements IConfig {
                 ACCESS_TOKEN_EXPIRES_IN: '24h',
                 REFRESH_TOKEN_EXPIRES_IN: '30d',
```
Comment (Vitaliy-Kalenichenko) [inline backend/src/common/config/config.package.ts:62]: Make sure that the cookie's lifetime expires at the same time as the refresh token's lifetime

**Review by Vitaliy-Kalenichenko — CHANGES_REQUESTED** [review]
Cookie signature - looks like it should be unique for each user
It's better to hardcode all expires_in values

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

**Comment by honkster-nazarii** [comment]
Is it really neccesary to update refresh token after `regenerateToken`?

**Comment by Vitaliy-Kalenichenko** [comment]
> Is it really neccesary to update refresh token after `regenerateToken`?

Are there any reasons not to do that?

**Pull request closed without merging**